draft-ietf-tls-tls13.md

```diff
@@ -4949,6 +4980,13 @@ server is not able to guarantee full uniqueness of the handshake
 state. See {{early-data-indication}} for one mechanism to limit
 the exposure to replay.
 
+TLS does not provide security for handshakes which take place after the peer's
+long-term secret (signature key or external PSK) is compromised. It therefore
+does not provide post-compromise security {{CCG16}}, sometimes also referred to
+as backwards or future security. This is in contrast to KCI resistance, which
+describes the security guarantees that a party may still have after its *own*
```

Comment:
The emphasis on "own" here will probably render strangely.  You should remove it.

Also, "may" implies permissiveness.  "a party has after" would work equally well and is more assertive (I hope that being definitive rather than equivocal here is OK, I tend to think that it's OK as long as P=~0).